strawberry/exceptions.py

```diff
@@ -220,3 +220,12 @@ def __init__(self):
         message = "`default_factory` must be a callable that requires no arguments"
 
         super().__init__(message)
+
+
+class InvalidCustomContext(Exception):
+    """Raised when a custom context object is of the wrong python type"""
+
+    def __init__(self):
+        message = "The custom context must be either a class\
+            that inherits from BaseContext or a dictionary"
```

Comment:
```suggestion
        message = (
            "The custom context must be either a class that inherits from "
            "BaseContext or a dictionary"
        )
```

Reply:
Avoid `\` whenever possible. If this were to get merged as-is, the string would look like:
```pycon
>>> message
'The custom context must be either a class            that inherits from BaseContext or a dictionary'
```
Your test, also using the `\`, misses this.

Reply:
Sorry about this one, I had it as something else previously but didn't realise they weren't equivalent. Will change!